src/geometries/ExtrudeGeometry.js

```diff
@@ -70,19 +84,8 @@ class ExtrudeGeometry extends BufferGeometry {
 
 			// options
 
-			const curveSegments = options.curveSegments !== undefined ? options.curveSegments : 12;
-			const steps = options.steps !== undefined ? options.steps : 1;
-			const depth = options.depth !== undefined ? options.depth : 1;
-
-			let bevelEnabled = options.bevelEnabled !== undefined ? options.bevelEnabled : true;
-			let bevelThickness = options.bevelThickness !== undefined ? options.bevelThickness : 0.2;
-			let bevelSize = options.bevelSize !== undefined ? options.bevelSize : bevelThickness - 0.1;
-			let bevelOffset = options.bevelOffset !== undefined ? options.bevelOffset : 0;
-			let bevelSegments = options.bevelSegments !== undefined ? options.bevelSegments : 3;
-
-			const extrudePath = options.extrudePath;
-
-			const uvgen = options.UVGenerator !== undefined ? options.UVGenerator : WorldUVGenerator;
+			const { curveSegments, steps, depth, extrudePath, UVGenerator: uvgen } = options;
```

Comment:
Can you please explain the `UVGenerator: uvgen` pattern? What exactly does it?

Reply:
Instead of linking to a doc page I was hoping you could say in one sentence what happens in the code.

Reply:
I interpret it like so: Since `uvgen` is not directly in `options`, it can't be unpacked from `options`.  So `UVGenerator` is unpacked and assigned to `uvgen`.

Reply:
This code does not use ES6 syntax, is it easy to accept?

Reply:
I leave this up to @mrdoob.

Sometimes, dump code is easier to read but I understand it's also a matter of taste. I approve code syntax related PRs only then when I know @mrdoob has approved the style in general.

Reply:
I still prefer the "dumb" code version yeah...